templates/dex-k8s-authenticator.yaml

```diff
@@ -58,5 +58,5 @@ spec:
           value: cGxhY2Vob2xkZXIK # b64 "placeholder" this will be updated by initcontainers
       initContainers:
       - name: initialize-dka-config
-        image: kubeaddons/addon-initializer:v0.0.6-alpha
+        image: kubeaddons/addon-initializer:v0.0.2
```

Comment:
what did it happen with `v0.0.1` :) ?

Reply:
lol we just kind of skipped it, because there was already technically a tag for that made.